packages/jest-runner/corpus.ts

```diff
@@ -56,6 +66,38 @@ const buildInputsDirectory = (
 	return path.join(root.dir, root.name, ...pathElements) + path.sep;
 };
 
+/**
+ * Returns the path to the corpus directory for the given test file and test descriptions.
+ */
+const buildCorpusDirectory = (
+	testPath: string,
+	testDescriptions: string[]
+): string => {
+	const root = findDirectoryWithPackageJson(path.parse(testPath));
```

Comment:
Move `path.parse` into the function and only pass `testPath` as argument? I think this could make the function simpler as only a string is required and no parameters are reassigned.